packages/browser/README.md

```diff
@@ -0,0 +1,170 @@
+# Backtrace Browser
+Backtrace error reporting tool for browser side JavaScript
+ 
+## Table of Contents
+1. [Integration Example](#integration-example)
+1. [Features](#features)
+    - [File Attachments](#file-attachments)
+    - [Metrics](#metrics)
+    - [Breadcrumbs](#breadcrumbs)
+    - [Callback Methods](#callback-methods)
+1. [Documentation](#documentation)
+
+## Integration Example
+```js
+// Import the BacktraceClient from @backtrace/browser with your favoriate package manager.
+import { BacktraceClient } from "@backtrace/browser";
+
+// BacktraceClientOptions
+const options = {
+    // Name of the website/application
+    name: "MyWebPage",
+    // Version of the website
+    version: "1.2.3",  
+    // Submission url
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
+}
+
+// Initialize the client
+const client = BacktraceClient.builder(options).build();
+
+// When an error occurs
+client.send(new Error("Something broke!"));
+
+```
+## Features
+
+### File Attachments
+***
+Backtrace has several ways in which files can be attached: when initalizing the BacktraceClient, updating the BacktraceClient, or  dynamically for specific reports.  When specifing attachments to the BacktraceClient, all files will be uploaded with each report.
+
+```js
+// Import the attachment types from @backtrace/browser with your favoriate package manager.
+import { BacktraceUint8ArrayAttachment, BacktraceStringAttachment  } from "@backtrace/browser";
+
+// BacktraceStringAttachment should be used for text object like a log file for instance
+var attachment1 = new BacktraceStringAttachment("logfile.txt", "This is the start of my log")
+
+// BacktraceUint8ArrayAttachment should be used for binary files
+var attachment2 = new BacktraceUint8ArrayAttachment("output.dmp", new Uint8Array(2));
```

Comment:
actually I'm not sure if this is the best example. You won't send minidump from web. Maybe instead we can send connection buffer or something like that?

Reply:
Okay.  Renaming the file name to `output_buffer`.